cairo/tests/utils/rust_compat.py

```diff
@@ -0,0 +1,48 @@
+from typing import Optional, Union
+
+from cairo_addons.vm import DictManager as RustDictManager
+from cairo_addons.vm import DictTracker as RustDictTracker
+from cairo_addons.vm import MemorySegmentManager as RustMemorySegmentManager
+from starkware.cairo.common.dict import DictManager, DictTracker
+from starkware.cairo.lang.vm.memory_segments import MemorySegmentManager
+from starkware.cairo.lang.vm.relocatable import MaybeRelocatable, RelocatableValue
+
+
+class SegmentsCompat:
```

Comment:
can you add documentation to this class to help readability
i understand from the context i have that we're juggling with both the cairovm in rust and cairovm py, and that as such we need to be able to manage segments interchangeably?